Winch/Serialization/Item/FishItemDataConverter.cs

```diff
@@ -2,26 +2,26 @@
 using Newtonsoft.Json.Linq;
 using UnityEngine;
 
-// ReSharper disable HeapView.BoxingAllocation
-
 namespace Winch.Serialization.Item;
 
 public class FishItemDataConverter : HarvestableItemDataConverter
 {
     private readonly Dictionary<string, FieldDefinition> _definitions = new()
     {
+        { "itemType", new(ItemType.GENERAL, null) },
+        { "itemSubtype", new(ItemSubtype.FISH, null) },
         { "minSizeCentimeters", new( 0f, o => float.Parse(o.ToString())) },
         { "maxSizeCentimeters", new( 0f, o => float.Parse(o.ToString())) },
         { "aberrations", new( new List<FishItemData>(), null) },
-        { "isAberration", new( false, null) },
+        { "isAberration", new( false, o => bool.Parse(o.ToString())) },
```

Comment:
What requires the bool string parsing? Is it not sufficient to rely on the json deserializer interpreting `"isAberration" : False,`?

Reply:
Honestly just putting it there to be safe- you are most likely entirely right and it’s redundant 👍